frontend/apps/vara-man/src/feature/tournament-game/components/modals/game-find.tsx

```diff
@@ -17,8 +17,21 @@ type GameFindModalProps = {
 	setIsOpenFindModal: (_: boolean) => void
 }
 
+const initialValues = {
+	username: "",
+};
+
+const validate = {
+	username: hasLength({ min: 2, max: 25 }, 'Username must be 2-25 characters long'),
+};
+
 export const GameFindModal = ({ findGame, setIsOpenFindModal }: GameFindModalProps) => {
-	const [username, setUsername] = useState("")
+	const form = useForm({
+		initialValues: initialValues,
```

Comment:
```suggestion
		initialValues,
```

No need to repeat the name